README.md

```diff
@@ -200,74 +196,68 @@ mymock/
     MakeFile
 ```
 
-Then on your config file, you will need configure `imposters` option like `.` because the `imposters file` are located in the
-same folder.
+Then in your config file, you will need to set the `-imposters` flag to `.` because the imposters folder is located in the same folder.
 
-Historically, the options `imposters_path`, `port`, `host` were mandatory when using a configuration file.
+Historically, the options `imposters_path`, `port`, `host` were mandatory when using a configuration file. However, since the last version, they are no longer needed, so you can simply override those options if you want to.
+Furthermore, in previous versions, the `imposters_path` option refered to the path where the app was launched, but in the last version this is relative to the location of the config file.
 
-However, since the last version, they are no longer needed, so you can simply override those options if you want to.
-Furthermore, the `imposters_path` option in previous version towards reference to the path where the app was launched, but in the last version it is relative on where the config file is.
-
-The option `cors` still being optional and its options can be an empty array.
+The option `cors` is still optional and its options can be an empty array.
 If you want more information about the CORS options, visit the [CORS section](#configure-cors).
 
-The `watcher` configuration field is optional, with this setting you can enable hot-reloads on imposter changes. Disabled by default.
+The `watcher` configuration field is optional. With this setting you can enable hot-reloads on imposter changes. Disabled by default.
 
-The `secure` configuration field is optional, with this setting you can run your server using TLS options with a dummy certificate, so to make it work with the `HTTPS` protocol. Disabled by default.
+The `secure` configuration field is optional. With this setting you can run your server using TLS options with a dummy certificate, so as to make it work with the `HTTPS` protocol. Disabled by default.
 
-The option `proxy` allow to configure the mock on a proxy mode, that means that you could configure an fallback urls, for all
-your calls or only the missing ones or none. More information: [Proxy Section](#prepare-killgrave-for-proxy-mode)
+The option `proxy` allows you to configure the mock in proxy mode. When this mode is enabled, Killgrave will forward any unconfigured requests to another server. More information: [Proxy Section](#prepare-killgrave-for-proxy-mode)
 
 ## How to use
 
 ### Configure CORS
 
-If you want to use `killgrave` on your client application you must consider to configure correctly all about CORS, thus we offer the possibility to configure it as you need through a config file.
+If you want to use `killgrave` in a client application you should consider configuring CORS.
```

Comment:
```suggestion
If you want to use `killgrave` from a client application you should consider configuring CORS.
```

"From" sounds a bit better for me because if you're testing a frontend application, you consume Killgrave exposed (mocked) API but not within the client itself (as a dep or something like that).